eng/packages/http-client-csharp/generator/Azure.Generator/src/AzureOutputLibrary.cs

```diff
@@ -44,16 +67,14 @@ private Dictionary<string, HashSet<OperationSet>> EnsureResourceDataMap()
             return result;
         }
 
-        private Dictionary<string, OperationSet> CategorizeClients()
+        private Dictionary<RequestPath, OperationSet> CategorizeClients()
```

Comment:
well this looks weird that we have this in OutputLibrary because `OperationSet` is actually an input. It would be nice if we could move this to input library - now we really have something on `InputLibrary`.
We could open an issue for this.

Reply:
Added TODO